docs/content/routing/providers/kubernetes-crd.md

```diff
@@ -1660,16 +1660,13 @@ or referencing TLS stores in the [`IngressRoute`](#kind-ingressroute) / [`Ingres
     
     spec:
       entryPoints:
-        - web
+        - websecure
       routes:
       - match: Host(`example.com`) && PathPrefix(`/stripit`)
         kind: Rule
         services:
         - name: whoami
           port: 80
-      tls:
```

Comment:
The TLS section should at least be empty to enable TLS.

Reply:
Got It. Updated in my last commit.